asax/lj.py

```diff
@@ -43,13 +47,9 @@ def on_atoms_changed(self):
         # clear neighbor list. force re-initialization.
         self._neighbors = None
 
-        # non-PBC atom passed - don't compute stress
-        if not all(self.atoms_cache.get_pbc()) and "stress" in self.implemented_properties:
-            self.implemented_properties.remove("stress")
-            return
-
-        if all(self.atoms_cache.get_pbc()) and "stress" not in self.implemented_properties:
-            self.implemented_properties.append("stress")
+        # non-PBC atom passed - disable stress computation
+        if not all(self.atoms_cache.get_pbc()) and self.stress:
```

Comment:
Shouldn't this be self.atoms now that you reverted?

Reply:
Never trust VS Code refactorings.